docs/6.x/changelog.md

```diff
@@ -37,6 +38,12 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 6.x Releases
 
+### 6.3.0 (December 18th, 2019)
+
+#### Improvements
+
+* Upgrade Kubernetes to `v1.17.0` ([#967](https://github.com/gravitational/gravity/pull/967), [planet#537](https://github.com/gravitational/planet/pull/537)).
```

Comment:
Don't forget the node role enhancement? and possibly hairpin removal mention

Reply:
I believe this version should be based off a branch that already has those changes, so I don't think it's necessary to mention this.

Reply:
Well 6.2.6 wasn’t a thing so no other release had it I believe